package.json

```diff
@@ -41,7 +27,5 @@
     ]
   },
```

Comment:
Sorry that I didn't mention this in the call, but I think we can remove this as well  👀